pkg/http_endpoint/v1/handlers/http_grafana.go

```
@@ -0,0 +1,337 @@
+				common.WriteErrorToResponse(w, http.StatusInternalServerError,
+					http.StatusText(http.StatusInternalServerError),
+					"provide Password in parameters")
+				return
+			}
+
+			// Create org user if no error
+			handler.CreateOrganizationUser(clients, res, w)
+		}
+	}
+}
+
+// DeleteOrganizationUser method deletes a user in an organization
+func DeleteOrganizationUser(clients *common.ClientContainer, handler common.HandlerInterface) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		organizationID := chi.URLParam(r, "organizationID")
+		ID, err := strconv.Atoi(organizationID)
+		if err != nil {
+			log.Logger.Error(err)
+			common.WriteErrorToResponse(w, http.StatusInternalServerError,
+				http.StatusText(http.StatusInternalServerError),
+				"provided organizationID is not not integer")
+			return
+		}
+		_, err = handler.GetUserID(clients, w, ID)
+		if err != nil {
+			return
+		}
+		err = handler.DeleteOrganizationUser(clients, w, ID)
+		if err != nil {
+			log.Logger.Error(err)
+			return
+		}
+	}
+}
+
+//GetOrganizationUser gets the user by organization
+func GetOrganizationUser(clients *common.ClientContainer, handler common.HandlerInterface) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		organizationID := chi.URLParam(r, "organizationID")
+		ID, err := strconv.Atoi(organizationID)
+		if err != nil {
+			log.Logger.Error(err)
+			common.WriteErrorToResponse(w, http.StatusInternalServerError,
+				http.StatusText(http.StatusInternalServerError),
+				"provided organizationID is not not integer")
+			return
+		}
+		_, err = handler.GetOrganizationID(clients, w, ID)
+		if err != nil {
+			return
+		}
+		orglist, err := handler.GetOrganizationUsers(clients, w, ID)
+		if err != nil {
+			common.WriteErrorToResponse(w, http.StatusInternalServerError,
+				http.StatusText(http.StatusInternalServerError),
+				"Internal server error occured")
+			return
+		}
+		orgs, _ := json.Marshal(orglist)
```

serialization error is not handled here